### PR TITLE
DBLike to Queryable

### PR DESCRIPTION
**Pull request by iamkevb**
Thought I'd pushed this one ages ago, but it's been sitting in my local branch.

### CODE REVIEW

File: Queryable.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package dblib
```
Comment (boourns) [inline Queryable.go]: uppercase filenames? is that typical in Go?

**Comment by boourns** [comment]
one question about filename casing otherwise :+1:

File: Queryable.go (hunk 1 above)
Comment (iamkevb) [reply Queryable.go]: nope. I haven't written Java in 10 years, but it's still the way I naturally write code.

**Comment by boourns** [comment]
lol what just happened

or should i say

qqqqwoooo

**Comment by boourns** [comment]
Ah, there we go, the other lowercase q

**Comment by iamkevb** [comment]
mac's file system isn't case sensitive. Git didn't see a difference between Queryable and queryable so I did a rename dance. Queryable -> qqqq -> queryable, then a squash. There's probably a better way.

**Comment by boourns** [comment]
assumed it was a good troll :D

**Pull request merged**